addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -83,6 +87,9 @@ const PopupWidget = publicWidget.Widget.extend({
      * @private
      */
     _onCloseClick: function () {
+        this.el.querySelectorAll('.media_iframe_video iframe').forEach(function(iframe) {
+            iframe.src = iframe.src;
```

Comment:
This is assuming the video will start with no sounds -> that's not an always truthy assumption AFAIK
1. Doing that is just restarting the video, with no sound, I don't think it is correct.
2. Also, but probably not very critical, the video will continue playing with sound if the admin/editor enter edit mode with the popup open -> the popup will be hidden by the editor code, but it won't be catched here obviously.

Maybe we should just have a BS4 open and hide event listener and the s_popup, and remove the src on hide (save it on data-src) and set the src on show (from its data-src)

@qsm-odoo Would sounds logical to you?

Reply:
@sga-odoo Can you try that solution?

Reply:
Hello @rdeodoo 

I have applied a solution according to your suggestion. https://github.com/odoo/odoo/compare/14.0...odoo-dev:14.0-website-popup-usability-imp-sga#diff-97412ce05b509aaab6bd0cb3aecd65f6f9420a0c3dbd66e97c89d83f85cfbaffR49

however, your second case is not still covered. When we click on the edit option, the editor is going to call destroy method of the snippet. Due to this BS4 hide event is not calling. 

let me know your thoughts on this.

Reply:
When re-thinking about this and testing the behavior(s), I actually noticed another problematic one -> The video plays in background before the modal show, the user might be missing most of the video because of that..
I pushed another fix that should handle that case too.